libs/tests-utils/ragstack_tests_utils/cassandra_container.py

```diff
@@ -12,6 +12,14 @@ def __init__(
         super(CassandraContainer, self).__init__(image=image, **kwargs)
         self.port = port
 
+        self.with_env(
```

Comment:
@nicoloboschi 

At least the `HEAP_NEWSIZE` and `MAX_HEAP_SIZE` were why this container wasn't starting for me locally. With these changes things work -- both `knowledge-store` and `langchain`.

Note for Langchain, we can run `poetry run pytest tests/integration_tests -k "[cassandra]` to run only those tests that use the local container (not sure if there is a way to make this a default, so it's easier to run without the astra environment setup).